exercises/tweet.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+import sys
+import redis
+import time
+import json
+
+HOST = "redis-13418.c8.us-east-1-4.ec2.cloud.redislabs.com"
+PORT = 13418
+PASS = "QCqZr7caZxWcUahORl0ReOg2JvqD8HZ4"
+POST_PREFIX = "posts:"
+FOLLOWEES_PREFIX = "followees:"
+USER = 'user'
+MESSAGE = 'message'
+TIME = 'time'
+
+
+def post(user: str, message: str, conn):
+    post_to_add = {MESSAGE: message, TIME: time.time()}
+
+    list_name = POST_PREFIX + user
```

Comment:
It would be more conventional in Python to do `list_name = f"{POST_PREFIX:user}" instead (and remove the colon from the `POST_PREFIX`).